docs/guide/packages.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Packages
+description: "UnoCSS Packages: available packages and what's included and enabled in unocss."
+outline: deep
+---
+
+# Packages
```

Comment:
At some point, I think VitePress should have some macros to import content from other MD files.

Reply:
https://vitepress.dev/guide/markdown#import-code-snippets